### PR TITLE
Fix bugs caused by using Decimate modifier

### PR DESCRIPTION
**Pull request by TikiBear**
Prevent export of Blender Decimate modifier from failing and corrupting Blender file!

### CODE REVIEW

File: korman/exporter/mesh.py [hunk 1]
```diff
@@ -178,7 +178,7 @@ def __exit__(self, type, value, traceback):
             for cached_mod in override["modifiers"]:
                 mod = bo.modifiers.new(cached_mod["name"], cached_mod["type"])
                 for key, value in cached_mod.items():
-                    if key in {"name", "type"}:
+                    if key in {"name", "type"} or (cached_mod["type"] == "DECIMATE" and key=="face_count"): # Decimate attribute face_count is read-only
```
Comment (Hoikas) [inline korman/exporter/mesh.py:181]: I saw this recently when making a sample Age for ametist and was wondering what was going on. I think we can make this fix a little more flexible by making a tuple of `(cached_mod["type"], key)` and comparing that against a sequence of known read-only attributes... In other words:

```python
readonly_attribs = {("DECIMATE", "face_count"),}
# ... (some code here) ...
if key in {"name", "type"} or (cached_mod["type"], key) in readonly_attribs:
    continue
```

That way if we find any more read-only attributes like `face_count`, it's fairly trivial to add them.

**Review by Hoikas — COMMENTED** [review]
Thanks for the pull request! It looks like you're using the github web UI to submit patches. In my experience, this is a little bit challenging to work with. You may want to investigate a git GUI such as [gitkraken](https://www.gitkraken.com/) or [fork](https://git-fork.com/) so you can work directly against the files on your computer.

File: korman/exporter/mesh.py (hunk 1 above)
Comment (TikiBear) [reply korman/exporter/mesh.py:181]: I like the tupling idea. I'll make an update. I'll go get a git GUI.

Comment (TikiBear) [reply korman/exporter/mesh.py:181]: Getting a git GUI turns out to be a problem for my old PC. Oh, well.

Comment (TikiBear) [reply korman/exporter/mesh.py]: I have updated per suggestion.

**Review by Hoikas — APPROVED** [review]
LGTM

**Pull request merged**